docs/content/en/docs/reference/clusterspec/nutanix.md

```diff
@@ -0,0 +1,322 @@
+ vcpuSockets: 2
+ vcpusPerSocket: 1
+---
+```
+
+## Cluster Fields
+
+### name (required)
+Name of your cluster `mgmt` in this example.
+
+### clusterNetwork (required)
+Specific network configuration for your Kubernetes cluster.
+
+### clusterNetwork.cniConfig (required)
+CNI plugin configuration to be used in the cluster. The only supported configuration at the moment is `cilium`.
+
+### clusterNetwork.cniConfig.cilium.policyEnforcementMode
+Optionally, you may specify a `policyEnforcementMode` of `default`, `always`, `never`.
+
+### clusterNetwork.pods.cidrBlocks[0] (required)
+Subnet used by pods in CIDR notation. Please note that only 1 custom pods CIDR block specification is permitted. This CIDR block should not conflict with the network subnet range selected for the VMs.
+
+### clusterNetwork.services.cidrBlocks[0] (required)
+Subnet used by services in CIDR notation. Please note that only 1 custom services CIDR block specification is permitted. This CIDR block should not conflict with the network subnet range selected for the VMs.
+
+### controlPlaneConfiguration (required)
+Specific control plane configuration for your Kubernetes cluster.
+
+### controlPlaneConfiguration.count (required)
+Number of control plane nodes
+
+### controlPlaneConfiguration.machineGroupRef (required)
+Refers to the Kubernetes object with Nutanix specific configuration for your nodes. See `NutanixMachineConfig` fields below.
+
+### controlPlaneConfiguration.endpoint.host (required)
+A unique IP you want to use for the control plane VM in your EKS Anywhere cluster. Choose an IP in your network range that does not conflict with other VMs.
+
+>**_NOTE:_** This IP should be outside the network DHCP range as it is a floating IP that gets assigned to one of
+the control plane nodes for kube-apiserver loadbalancing. Suggestions on how to ensure this IP does not cause issues during cluster 
+creation process are [here]({{< relref "../nutanix/nutanix-prereq/#prepare-a-nutanix-environment" >}}).
+
+### controlPlaneConfiguration.taints
+A list of taints to apply to the control plane nodes of the cluster.
+Replaces the default control plane taint, `node-role.kubernetes.io/master`. The default control plane components will tolerate the provided taints.
+ 
+Modifying the taints associated with the control plane configuration will cause new nodes to be rolled-out, replacing the existing nodes.
+ 
+>**_NOTE:_** The taints provided will be used instead of the default control plane taint `node-role.kubernetes.io/master`.
+Any pods that you run on the control plane nodes must tolerate the taints you provide in the control plane configuration.
+> 
+
+### controlPlaneConfiguration.labels
+A list of labels to apply to the control plane nodes of the cluster. This is in addition to the labels that EKS Anywhere will add by default.
+ 
+Modifying the labels associated with the control plane configuration will cause new nodes to be rolled out, replacing the existing nodes.
```

Comment:
Node taints and labels aren't tested yet and most likely won't work with CAPX until we add support for them explicitly. I don't think we should mention this for now